elements/howto-checkbox/howto-checkbox.js

```diff
@@ -0,0 +1,161 @@
+/**
+ * A `HowToCheckbox` represents a boolean option in a form. The most common type
+ * of checkbox is a dual-type which allows the user to toggle between two
+ * choices -- checked and unchecked.
+ */
+(function() {
+  /**
+   * Define key codes to help with handling keyboard events.
+   */
+  const KEYCODE = {
+    SPACE: 32,
+  };
+
+  /**
+   * The `HowToCheckbox` exposes a single `checked` attribute/property for
+   * toggling its state. Changes to the `checked` property will also be
+   * reflected to an `aria-checked` attribute. Similarly, the `disabled`
+   * property is reflected to an `aria-disabled` attribute. This controls
+   * whether the element is operable or not.
+   */
+  class HowToCheckbox extends HTMLElement {
+    static get observedAttributes() {
+      return ['checked', 'disabled'];
```

Comment:
ah before I forget...

An important reason to reflect is for styling. A native checkbox has a `:checked` pseudo class, but there isn't a similar pseudo class for `disabled`. I wonder if that's why this behavior was chosen?

In our case, we don't have access to any pseudo classes so we need to reflect if folks want to style based on element state.